src/model/trust.model.ts

```diff
@@ -48,7 +48,7 @@ export interface TrustReviewStatus {
 export interface TrustIndividual {
   id?: string;
   ch_references?: string;
-  type: RoleWithinTrustType | undefined;
```

Comment:
i can see we are removing the undefined option, could we still have trust individuals on the database that don't have a type defined? as that might cause an issue if they try and update them.

Reply:
thinking about it, i guess it can still be undefined if the type isn't mapped, so should be ok.